package.json

```diff
@@ -17,7 +17,10 @@
     "verbose": true,
     "roots": [
       "<rootDir>/src/"
-    ]
+    ],
+    "moduleNameMapper": {
+      "\\.(css|scss)$": "<rootDir>/src/js/tests/__mocks__/styleMock.js"
```

Comment:
if you want to test component that imports some css with jest it will fail, because jest it's handling only javascript and you need to mock any other file types